src/components/molecules/plan/plan.scss

```
@@ -1,103 +1,77 @@
 @use "../../variables" as *;
 
-.plan {
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  align-items: center;
-
-  @media (max-width: $breakpoint-md) {
-    flex-direction: column;
-    margin-bottom: 24px;
-  }
-
-  & > .plan-daytime {
-    width: 100%;
-  }
-
-  & > .plan-standard {
+.plan-set {
+  .plan {
+    text-align: center;
     width: 100%;
 
-    ul {
-      border: 5px solid $orange_bistre;
-
-      li:nth-child(1) {
-        height: 59px;
-        background-color: $orange_bistre;
-        padding: 15px 8px;
-      }
-    }
-  }
-
-  ul {
-    list-style: none;
-    padding: 0px;
-    margin: 0px 12px;
-
     @media (max-width: $breakpoint-md) {
       margin-bottom: 24px;
     }
 
-    li {
-      padding: 20px 8px;
-      text-align: center;
-      margin: -1px 0px;
-      border: 1px solid $gray_gainsboro;
-    }
-
-    li:nth-child(1) {
-      height: 69px;
-      color: $white;
-      font-size: $fontsize-l;
-      font-weight: 700;
-      line-height: 29px;
+    .plan-text {
       background-color: $orange_zorba;
-      margin: 0px;
-      border: none;
+      padding: 20px 0;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+
+      h3 {
+        color: $white;
+        margin-bottom: 0;
+        font-size: $fontsize-l;
+      }
     }
 
-    li:nth-child(2) {
-      height: 142px;
-      background-color: $gray_desertStorm;
+    .plan-wrap {
+      border: 1px solid $gray_gainsboro;
 
       .plan-price {
-        display: flex;
-        flex-direction: row;
-        justify-content: center;
-        align-items: flex-end;
-        color: $orange_bistre;
-        font-weight: 700;
-        gap: 4px;
-        margin: 0px 0px 16px;
+        background-color: $gray_desertStorm;
+        padding: 24px 0;
+        height: 142px;
 
-        .plan-price-mark {
-          font-size: $fontsize-l;
-          line-height: 29px;
-        }
-
-        .plan-price-yen {
+        h3 {
           font-size: $fontsize-xxl;
-          line-height: 58px;
+          font-weight: 700;
+          color: $orange_bistre;
+          margin-bottom: 16px;
+
+          .yen {
+            font-size: $fontsize-l;
+            margin-right: 4px;
+          }
         }
 
-        .plan-price-mark,
-        .plan-price-yen {
-          margin: 0px;
+        .monthly {
+          font-size: $fontsize-s;
+          font-weight: 400;
+          color: $black;
         }
       }
 
-      .text {
-        color: $black;
-        font-size: $fontsize-s;
-        margin: 0px;
+      .plan-detail {
+        .lists {
+          padding: 0px;
+          margin: 0;
+          font-size: $fontsize-s;
+
+          .list-item {
+            list-style: none;
+            padding: 20px;
+            border: 1px solid $gray_gainsboro;
+            border-top: none;
```

横が二重になっているので以下の修正お願いします
直ってるかの確認も忘れずにお願いします
```suggestion
            border-bottom: 1px solid $gray_gainsboro;
```

---

last-childのborder-bottomをnoneにすると一番下が二重になっているのが解消されると思います